### PR TITLE
Fix not existing props values on donate trees

### PR DESCRIPTION
**Pull request by mahmed0715**
@norbertschuler please check Fix https://github.com/Plant-for-the-Planet-org/treecounter-app/issues/1848 https://github.com/Plant-for-the-Planet-org/treecounter-app/issues/1843 https://github.com/Plant-for-the-Planet-org/treecounter-app/issues/1846

### CODE REVIEW

**Comment by norbertschuler** [comment]
I can’t say if with this fix all the issues with missing data are fixed as I couldn’t say for https://github.com/Plant-for-the-Planet-org/treecounter-app/pull/1838. This is just a patch for this special case which I can test tomorrow, but the software needs a general refactoring and check of all use cases of the donation process in order to be seen as stable again. Currently I regard this part of our software as unstable.

**Comment by mahmed0715** [comment]
just fixing adhoc for now, we need huge refactoring and re engineering the donation process.. @norbertschuler

**Comment by mahmed0715** [comment]
i see the payment failed because of this api https://www.trilliontreecampaign.org/public/v1.3/en/paymentInfo/KW64UGK5XA payment info null

File: app/components/DonateTrees/index.native.js [hunk 1]
```diff
@@ -148,7 +148,9 @@ export default class DonateTrees extends React.PureComponent {
 
   // open your gateway
   openGateWay = async url => {
+    const localUrl = 'http://localhost:8081' + url;
```
Comment (sagararyal) [inline app/components/DonateTrees/index.native.js]: @mahmed0715 do we need this on the repo?

Comment (mahmed0715) [reply app/components/DonateTrees/index.native.js]: nope, i tried to have the actual url we are redirecting to web, which is not actually debuggable from mobile app.. 
now i removed all those ..
@sagararyal

**Pull request merged**